src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/HearingDuration.java

```diff
@@ -699,13 +697,11 @@ public enum HearingDuration {
     private final Integer durationInterpreter;
     private final Integer durationPaper;
 
-    private static final Map<BenefitCode, Map<Issue, HearingDuration>> BY_BENEFIT = new HashMap<>();
-    private static final  Map<Issue, HearingDuration> BY_ISSUE = new HashMap<>();
+    private static final  Map<Integer, HearingDuration> BY_QUERY = new HashMap<>();
 
     static {
         for (HearingDuration e: values()) {

```

Comment:
Optional: I personally don't like single character variable names but it's your preference, would call this `hearingDuration`